pkg/kubernetes/client.js

```diff
@@ -805,7 +808,7 @@ define([
         /* The watch objects we have open */
         self.watches = { "events": new KubernetesWatch("events", handle_event, handle_removed) };
         [ "nodes", "pods", "services", "replicationcontrollers",
-          "namespaces", "endpoints" ].forEach(self.include);
+          "namespaces", "endpoints", "deploymentconfigs" ].forEach(self.include);
```

Comment:
Could we have the TopologyCtrl and DetailsCtrl do ```client.include("deploymentconfigs")```

They aren't used on every page after all...